h1st/model/remote_model.py

```diff
@@ -0,0 +1,157 @@
+from typing import Dict, Tuple
+import requests
+from .predictive_model import PredictiveModel
+import pandas as pd
+
+
+class RemoteModel(PredictiveModel):
+    """
+    Class for making inference calls with models served via the Aitomatic Wed API
+
+    Ex: 
+    model = RemoteModel(API_TOKEN, "MyModelName").load_params()
```

Comment:
in the new version, we will use `load` function officially to replace this `load_params` function. temporary we still keep this function for backward compability